beacon/goclient/signing.go

```diff
@@ -11,6 +14,58 @@ import (
 	"github.com/pkg/errors"
 )
 
+func (gc *goClient) computeVoluntaryExitDomain(ctx context.Context) (phase0.Domain, error) {
+	specs, err := gc.client.Spec(gc.ctx, &api.SpecOpts{})
+	if err != nil {
+		return phase0.Domain{}, errors.Wrap(err, "failed to obtain specs")
+	}
+	if specs == nil {
+		return phase0.Domain{}, fmt.Errorf("failed to obtain specs")
+	}
+	if specs.Data == nil {
+		return phase0.Domain{}, fmt.Errorf("failed to obtain specs")
```

Comment:
Perhaps a bit different text would help to understand what exactly is nil if this error appears in logs